src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.hpp

```diff
@@ -13,6 +13,8 @@
 #include "ie_system_conf.h"
 
 namespace CPUTestUtils {
+    static std::string any_type = "any_type";
```

Comment:
```suggestion
    static const std::string any_type = "any_type";
```

Reply:
Moreover, it is better to define it in cpp.

Reply:
The definition and declaration changed according to offline discussion